spine-ios/Sources/Spine/Spine.Generated.swift

```diff
@@ -26,8 +26,10 @@ public final class TransformConstraintData: NSObject {
     public var bones: [BoneData] {
```

Comment:
Do you think it makes sense to simplify the code-gen so we call `compactMap` directly on the buffer, would this work?

```swift
public var bones: [BoneData] {
    let num = Int(spine_skeleton_data_get_num_bones(wrappee))
    guard let ptr = spine_skeleton_data_get_bones(wrappee) else { return [] }
    let buffer = UnsafeBufferPointer(start: ptr, count: num)
    return buffer.compactMap { $0.flatMap { .init($0) } }
}
```

It looks like we are copying the `UnsafeBufferPointer`s content two times. Once with Array(buffer) and once with the `.compactMap`?

Reply:
Yes, you're right. Creating an array with `Array(buffer)` indeed involves a memory copy. Thanks for pointing that out!